coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = vars(result)
```

Comment:
I guess it's fine, if you see even the `severity_str` is getting overwritten for every iteration ^ Like I mentioned in the other comment, it _is_ required.